From 7e7e0dde0495e105bb717418745cc24edd051887 Mon Sep 17 00:00:00 2001
Message-Id: <7e7e0dde0495e105bb717418745cc24edd051887.1549965172.git.liu.denton@gmail.com>
In-Reply-To: <cover.1549965172.git.liu.denton@gmail.com>
References: <0b62f65a8016c41b96aa6caead366e1ed9dda333.1549944164.git.liu.denton@gmail.com>
	<cover.1549965172.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Tue, 12 Feb 2019 01:28:35 -0800
Subject: [PATCH v2 1/3] git-remote.txt: be specific about default behavior
To: git@vger.kernel.org
Cc: pclouds@gmail.com

Instead of beating around the bush, we directly specify the subcommand
that is called by default when none are provided. This way, a user will
know where to look for more information.
---
 Documentation/git-remote.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/git-remote.txt b/Documentation/git-remote.txt
index 0cad37fb81..a2e685818b 100644
--- a/Documentation/git-remote.txt
+++ b/Documentation/git-remote.txt
@@ -41,8 +41,8 @@ OPTIONS
 COMMANDS
 --------
 
-With no arguments, shows a list of existing remotes.  Several
-subcommands are available to perform operations on the remotes.
+With no arguments, the default command is 'show'.  Several subcommands are
+available to perform operations on the remotes.
 
 'add'::
 
-- 
2.20.1.522.gd8785cdd01

